integration/integration.test.ts

```diff
@@ -1,74 +1,106 @@
-import {
-  newAuthorSummary,
-  newCurrentAuthorResponse,
-  newGetAuthorSummariesResponse,
-  newSaveAuthorResponse,
-} from "./graphql-types";
+import * as SingleFile from "./singleFile/graphql-types.generated";
```

Comment:
This pattern is inspired by the integration tests in https://github.com/homebound-team/graphql-typescript-factories/blob/main/integration/integration.test.ts

With a few improvements (like putting the generated code in its own folder and `.gitignore`-ing it).

It's much easier to review this changeset with whitespace turned off: https://github.com/homebound-team/graphql-typescript-response-factories/pull/33/files?w=1